audb/core/load.py

```diff
@@ -1093,10 +1093,11 @@ def load(
         num_workers: number of parallel jobs or 1 for sequential
             processing. If ``None`` will be set to the number of
             processors on the machine multiplied by 5
-        timeout: maximum wait time if another thread or process is already
-            accessing the database. If timeout is reached, ``None`` is
-            returned. If timeout < 0 the method will block until the
-            database can be accessed
+        timeout: maximum wait time in seconds
```

Comment:
Would you mention what is being waited for? My understanding is that you want to acquire the lock yourself and are willing to wait for `timeout`  before you give up.

Reply:
The full text is:

```
        timeout: maximum wait time in seconds
            if another thread or process is already
            accessing the database.
            If timeout is reached,
            ``None`` is returned
```

So, we are waiting that we can get access to the database. This needs to be blocked if another user is currently accessing it, as this means the cache might be not fully filled yet.

What would be your suggestion as additional text?

Reply:
>> So, we are waiting that we can get access to the database. 

Yes, and you get access to the database when the lock is removed.  When reading the first line  "maximum wait time in seconds"  I expect something like "before"  - for example before giving up my attempt to acquire the lock (and hence giving up the attempt to get database access). So for me the "if" is a little confusing.

Reply:
The `if` indicates that we only need to wait if another thread or process is accessing the database. I guess the original idea was to avoid terms like lock or lock file. But as we have the problem of lock files that are not deleted, it might indeed be better to change the text. How about:

```
        timeout: maximum time in seconds
            before giving up acquiring a lock to the database cache folder.
            ``None`` is returned in this case
```

Reply:
For me it becomes clearer like this.